### PR TITLE
[FIX] website_*: replace manual tour with utils

### PR DESCRIPTION
**Pull request by divy-odoo**
In this PR, 
Replace the manual tour steps with existing tour utils.

task-3279232

### CODE REVIEW

File: addons/website/static/tests/tours/website_form_editor.js [hunk 1]
```diff
@@ -629,10 +571,8 @@ odoo.define('website.tour.form_editor', function (require) {
         }, {
             content: "Choose 'not equal to' comparator",
             trigger: 'we-button[data-select-data-attribute="!selected"]',
-        }, {
-            content: 'Save the page',
-            trigger: 'button[data-action=save]',
         },
+        ...wTourUtils.clickOnSave(),
```
Comment (rdeodoo) [inline addons/website/static/tests/tours/website_form_editor.js:575]: remove `editor_enable` below

File: addons/website/static/tests/tours/website_form_editor.js [hunk 2]
```diff
@@ -501,11 +454,7 @@ odoo.define('website.tour.form_editor', function (require) {
         ...addCustomField("char", "text", "``", false),
         ...addCustomField("char", "text", "Same name", false),
         ...addCustomField("char", "text", "Same name", false),
-        {
-            content: 'Save the page',
-            trigger: 'button[data-action=save]',
-            run: 'click',
-        },
+        ...wTourUtils.clickOnSave(),
```
Comment (rdeodoo) [inline addons/website/static/tests/tours/website_form_editor.js:457]: remove editor_enable below

File: addons/website/static/tests/tours/website_form_editor.js [hunk 3]
```diff
@@ -486,11 +443,7 @@ odoo.define('website.tour.form_editor', function (require) {
             run: () => null, // it's a check
         },
         ...wTourUtils.clickOnEditAndWaitEditMode(),
-        {
-            content: 'Click on the submit button',
-            trigger: 'iframe .s_website_form_send',
-            run: 'click',
-        },
+        wTourUtils.clickOnElement('submit button', 'iframe .s_website_form_send'),
```
Comment (rdeodoo) [inline addons/website/static/tests/tours/website_form_editor.js:446]: revert (and remove `run: 'click'`)

File: addons/website/static/tests/tours/website_form_editor.js [hunk 4]
```diff
@@ -381,10 +345,7 @@ odoo.define('website.tour.form_editor', function (require) {
             trigger: 'we-input[data-attribute-name="value"] input',
             run: "text prefilled",
         },
-        {
-            content: "Save the page",
-            trigger: "button[data-action=save]",
-        },
+        ...wTourUtils.clickOnSave(),
```
Comment (rdeodoo) [inline addons/website/static/tests/tours/website_form_editor.js:348]: remove editor_enable below

File: addons/website/static/tests/tours/website_form_editor.js [hunk 5]
```diff
@@ -331,13 +299,9 @@ odoo.define('website.tour.form_editor', function (require) {
             },
         },
         // Edit the submit button using linkDialog.
+        wTourUtils.clickOnElement('submit button to show edit popover', 'iframe .s_website_form_send'),
+        wTourUtils.clickOnElement('Edit Link in Popover', 'iframe .o_edit_menu_popover .o_we_edit_link'),
```
Comment (rdeodoo) [inline addons/website/static/tests/tours/website_form_editor.js:303]: revert

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Hello and thank you for your work.

Note that we would prefer if you ask before taking an internal website task.
There is tasks for which we already have discussed it at the office and have a clear idea of the scope and what should be done.
Or simply tricky tasks that we would like to keep here so we can easily discuss it IRL during development.

About this PR, I did not comment all lines, so please apply comments to all the PR and not just where I commented.

Basically:
- Revert the `clickOnElement` diff
- Remove the `editor_enable` checks when you introduced a `clickOnSave`
- Move in a separate commit the diff which is about creating new **local** utils (which is not part of tour_utils but just a variable in a local file used in this file only). This is great but this is not part of the task and just nitpicking cleaning.
Using the utils from tour_utils (initial goal of the task) on the other hand is not nitpicking: those utils have been battle tested and are robust against race condition, it's why we want to use those. It also means that when we will fix those later, every tests will benefit from it.
- Either keep in 16.0 only the existing `tour_utils` usage diff and move the rest in master, or move everything in master (as per the task description...)

**Comment by divy-odoo** [comment]
Hello @rdeodoo, Changes are applied according to your suggestions.

- For now, just keeping existing tour utils diff only in this PR.

**Review by rdeodoo — APPROVED** [review]
@robodoo r+ Ty for your work

I have push [this diff](https://github.com/odoo/odoo/compare/db6f33817605e8fdbffe8b156f117ecee0279e44..89c8cd6c88761d1995dc01524145c3e73d44eef2) to apply the review points so it can directly be merged.

Hopefully it's still green.

Did you check enterprise too?

**Review by Feyensv — APPROVED** [review]

**Comment by rdeodoo** [comment]
@robodoo r+

@divy-odoo You [push forced](https://github.com/odoo/odoo/compare/89c8cd6c88761d1995dc01524145c3e73d44eef2..125e3b1d9c39397213d19848a47d5f6d1ca58053) after I had r+ and told you I made the final changes myself, which basically just silently canceled the r+
I guess you tried to fix the red upgrade build by rebasing the branch? Please mention it when you do that so we can keep track of things going on on PRs.

**Comment by divy-odoo** [comment]
Oh, Sorry it's my bad @rdeodoo . I forgot to comment. :slightly_smiling_face:

Yes, I just rebased the branch.

**Pull request closed without merging**